src/index.js

```diff
@@ -91,38 +91,40 @@ MyArray.prototype.filter = function(callback, thisArg) {
 };
 
 MyArray.prototype.reduce = function(...rest) {
```

Comment:
You have changed the reduce method, but there is no information about it in the commit.